lib/src/native/rtc_data_channel_impl.dart

```diff
@@ -51,7 +51,7 @@ class RTCDataChannelNative extends RTCDataChannel {
     final Map<dynamic, dynamic> map = event;
     switch (map['event']) {
       case 'dataChannelStateChanged':
-        _dataChannelId = int.tryParse(map['id']);
+        _dataChannelId = int.tryParse(map['id'].toString());
```

Comment:
But `tryParse` could return `null`, so it seems that there must be some additional condition about that.

Reply:
I'm not sure if it's a platform-specific exception, the type of id should be enforced at the native layer, I think we have to fix it at the native layer

Reply:
I mean, the `id` must exist in `dataChannelStateChanged` or other `events`, and the type must be `int`, so as to avoid exceptions,